src/cosmic_ray/work_item.py

```diff
@@ -74,6 +74,7 @@ class ResolvedMutationSpec(MutationSpec):
     "A MutationSpec with the location of the mutation resolved."
     start_pos: Tuple[int, int]
     end_pos: Tuple[int, int]
+    operator_args: Optional[Dict] = None
```

Comment:
As I mentioned elsewhere, I think this field should be on `MutationSpec`, not `ResolvedMutationSpec`.